### DISCUSSION TITLE
Using delve to debug

### DISCUSSION THREAD

**Discussion by gsovereignty** [wailsapp/wails#2261] [General]
Has anyone successfully used delve to debug something built with wails?

**Comment by leaanthony** [comment]
Yes. What's the problem?

**Reply by gsovereignty** [reply]
Wasn't expecting a reply that quickly, thanks!

I usually just use `dlv debug <main go file or dir>` to debug my projects.

But I'm not sure how to do it with a wails project where you need wails to build everything with `wails dev`. What's your workflow or how do you usually do it?

**Reply by gsovereignty** [reply]
While you're here, is there a faster way to regenerate the typescript bindings? I've just been running `wails dev` every time

**Reply by shukpa** [reply]
any update on this thread?

**Reply by leaanthony** [reply]
What do you want to know?

**Reply by shukpa** [reply]
hi Lea, I wanted to understand an easy way to debug golang code with a wails app. `dlv` is the tool that is supported by VSCode for golang debugging. I also saw an IDE section in the wails docs - https://wails.io/docs/guides/ides/

Will try the above out as well to see if it helps debugging.

**Reply by leaanthony** [reply]
If you are using vscode or goland, I'd recommend creating a new project with the `ide` flag and have a look at the ide specific directory. For vanilla delve, you can follow the manual build guide and use delve like you normally would. We need a better guide on this

**Comment by hao-lieu** [comment]
are there instructions for debugging in goland?
